nlp/Dockerfile

```diff
@@ -8,7 +8,7 @@ COPY notebook_config.yml /notebook_config.yml
 
 # update env "birdy"
 # use umask 0000 so that package files for the updated environment are usable by the user for the jupyter-conda-extension
-RUN umask 0000 && conda env update -f /environment.yml
+RUN umask 0000 && mamba env update -f /environment.yml
```

Comment:
Where is mamba coming from? I've tried calling it from `birdhouse/pavics-jupyter-base:0.3.0` but it's not found.